### ISSUE TITLE
New home page to reach the various communities

### ISSUE THREAD

**Issue by Mte90** [mozilla/discourse#275]
Now with the community portal there are sections for the various groups inside Discourse but is not easy to reach team.
As example I have a bookmark for the section but instead as user I am expecting that the home page include links to reach the various sections like for locales as example.

This can be part of a way to help communities to migrate from their forums to Discourse but right now is very difficult to use this page and not very welcome.
The order of the section list change based on the activity but I think that we need to have some of them at the top.

**Comment by nukeador** [comment]
The homepage to discover, search and filter mozilla communities and groups should be this one:

https://community.mozilla.org/groups/

This will allow you to get all the information, including their discourse.

**Comment by Mte90** [comment]
Yes but if an user ask me what is the forum of Mozilla Italia as example, I have to say search on community portal the group (if there aren't duplicates or fake groups because everyone can create them), scroll the page to the discourse section, click the first link on that and you are in.

As approach is not comfortable if someone want just to have the discourse forum. At the same time the discourse website doesn't have any information about the community portal in home like a banner so for anyone they are like 2 different platform.
